usage_windows.go

```diff
@@ -62,7 +62,7 @@ func platformCommandLineParameters() string {
     --service-name SERVICE-NAME             The name that the Windows service should be
                                             installed under. [default: Generic Worker]
     --working-directory DIRECTORY           The working directory the Generic Worker
-                                            service will use. [default: C:\Windows\system32]`
+                                            service will use.`
```

Comment:
Can we determine the system default at runtime and include it in the usage text?

Reply:
I think defaulting to the parent directory of the generic worker binary is a reasonable default. It's also trivial to implement in a cross platform manner.

Reply:
That's fine, but we should describe what the default is in the docs.